src/core/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,37 @@
+import { Props } from '@guardian/src-helpers';
+import React, { ReactElement } from 'react';
+import { accordion } from './styles';
+
+export interface AccordionProps extends Props {
+	/**
+	 * Determine whether to display the "show" and "hide" labels that toggle the collapsing and expanding of the menu. When set to `true`, the label will no longer appear. It may be useful to toggle this flag when there is limited space.
```

Comment:
Are there consequences of adding manual line breaks to this comment to make it fit on one screen? I'm guessing it will botch how it appears in Storybook

Reply:
fixed 26064bb